src/components/TopBar/TopBarMenu.vue

```diff
@@ -251,10 +239,11 @@ export default {
 		},
 	},
 
+	emits: ['open'],
+
```

Comment:
In my opinion, the name `open` of `TopBarMenu` doesn't describe what exactly happened, what was supposed to open (or was opened) in the menu with many actions. I'd name it `open-breakout-rooms-editor` or `edit-breakout-rooms`.